### PR TITLE
Backport #1883 -- CUMULUS-2168

### PR DESCRIPTION
**Pull request by Jkovarik**
**Summary:** Summary of changes

Addresses [CUMULUS-2168: Develop amazing new feature](https://bugs.earthdata.nasa.gov/browse/CUMULUS-2168)

## Changes

* Back-porting #1883 to the 2.0.x release branch.  Should not be merged until that one is reviewed. 

## PR Checklist

- [x] Update CHANGELOG
- [x] Unit tests
- [x] Adhoc testing
- [ ] Integration tests

### CODE REVIEW

File: packages/api/es/collections.js [hunk 1]
```diff
@@ -27,19 +28,35 @@ class Collection extends BaseSearch {
...
-                  count: {
+                  version: {
                     terms: {
-                      field: 'status'
+                      field: 'version'
+                    },
+                    aggs: {
+                      stats: {
+                        children: {
+                          type: 'granule'
```
Comment (Jkovarik) [inline packages/api/es/collections.js:50]: Reviewers - note I didn't include the size de-limiter from #1883.    That's up for debate, although given this is fixing a prod issue safer is *probably* better.     That's up for debate in this case.

Comment (markdboyd) [reply packages/api/es/collections.js:50]: I agree with this decision

Comment (laurenfrederick) [reply packages/api/es/collections.js:50]: So I would say that is good for 2.0.6. I'm guessing it's passing the tests then that I put in for >10 granules on master?? (see https://github.com/nasa/cumulus/pull/1870 and you can probably just close that one or I can once you confirm)

**Review by laurenfrederick — APPROVED** [review]

File: packages/api/es/collections.js (hunk 1 above)
Comment (Jkovarik) [reply packages/api/es/collections.js:50]: @laurenfrederick  if you mean #1883 and the units introduced in #1821 then yes, the master PR is passing all tests.

**Pull request merged**